### DISCUSSION TITLE
Have Pretix & Pretalx Communicate on Speaker Tickets

### DISCUSSION THREAD

**Discussion by theschweitz** [pretix/pretix#2960] [Ideas]
Right now, many things have to take place between when a Speaker buys a ticket (so they can attend the conference regardless) and when they are notified that their Talk Submission has been accepted. This requires the organizer to cross-check speaker data with all existing orders, to ensure that no speaker has multiple tickets in their name.

If we're lucky, Speakers will email our support team to let us know that they had previously acquired a ticket that they do not need now that they are an accepted Speaker. 
But as we saw in our last conference, we had many Speakers show up to Register with a General Admission ticket, and then alert us after scanning that they were actually a Speaker. We then had to verify this by checking our Speakers list & changing the Product Type to Speaker Ticket so our figures would be accurate.

We would love the ability for Pretalx & Pretix to work together & to automatically create or update existing tickets for confirmed speakers.

**Comment by raphaelm** [comment]
Automatically creating is one thing, that sure would be nice. But it would lead to even mote duplicates. Having pretalx search and modify existing tickets sounds like a nightmare of possible edge-case confusion scenarios to me, though.
